INSTALL.md

```diff
@@ -61,3 +61,15 @@ You can verify that the installation is working by running the toy example
 ```
 After the script completed, you should be able to find some metric summaries and plots in the **examples/toy** folder.
 
+It's also possible to download some pre-processed Quijote data from ...
+
```

Comment:
@maho3 Can you maybe add where one can find the quijote data

Reply:
Done!

Reply:
Great thanks!